sdk/src/main/java/nl/rabobank/gict/payments_savings/omnikassa_frontend/sdk/model/response/OrderStatusResponse.java

```diff
@@ -0,0 +1,71 @@
+package nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.response;
+
+import kong.unirest.json.JSONObject;
+
+import nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.model.Money;
+
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.List;
+import java.util.Spliterator;
+import java.util.stream.Collectors;
+import java.util.stream.StreamSupport;
+
+import static nl.rabobank.gict.payments_savings.omnikassa_frontend.sdk.utils.CalendarUtils.stringToCalendar;
+
+public class OrderStatusResponse {
```

Comment:
OrderStatusResponse should only contain 'order' filed. We should create a separate class and name it OrderStatusResult which will contain all of fields mentioned in this class. After that we can refer 'order' filed to be of type  OrderStatusResult.

Reply:
Done.